lib/models/BaseAsset.ts

```diff
@@ -14,6 +14,10 @@ export class BaseAsset {
     this._id = _id || BaseAsset.id(content.type, content.model, content.reference);
 
     this._source = content;
+
+    if (! Array.isArray(this._source.measures)) {
+      this._source.measures = [];
+    }
```

Comment:
It can be anything else, a string, an object etc because a developer or user can modify this document.  

Unfortunately it can break the rest of the plugin and it has no sense to put anything else than an array, that's why I'm enforcing it like this